lib/multiaddr.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @param {string} addr Multiaddr, e.g. `/ip4/127.0.0.1/tcp/80/http`
+ * @returns {string} Parsed URI, e.g. `http://127.0.0.1:80`
+ */
+export function multiaddrToHttpUri (addr) {
```

Comment:
why did you choose Uri over Url here?

Reply:
I don't remember, I am happy to change the name to Url.

Reply:
In my understanding it's a URL, since it contains the protocol and points at a specific location